src/server/serverUtils/contentLoader.ts

```diff
@@ -293,7 +293,11 @@ export class ContentLoader extends Disposable {
 		let contentLength = 0;
 
 		while (i < workspaceDocuments.length) {
-			if (PathUtil.PathEquals(readPath, workspaceDocuments[i].fileName)) {
+			let docFileName = workspaceDocuments[i].fileName;

```

Comment:
Untitled files have a filename/path that doesn't start with `/`, but codeblocks do. `readPath` has the slash trimmed off.